### PR TITLE
chore(dev): updating proxy url and minor cleanup

### PR DESCRIPTION
**Pull request by staceysv**
questions/todos:

- [ ] I saw no places to change the base url in the enterprise mode proxy based on the logic (enterprise urls will be different), is that correct/expected/am i missing something?
- [ ] old url still works for proxy calls, is that ok/expected?

- update proxy url
- fix curl cell with proper syntax so it runs
- minor wording
- clean up headings so it's easier for readers to orient
- remove last cell until we can resolve behavior between weave sdk and dev env authentication

### CODE REVIEW

**Comment by jlzhao27** [comment]
can you update the URLs here are well: weave/panels_py/instructions/panel_llm_monitor.md

**Comment by jlzhao27** [comment]
1. yes enterprise mode should not be changed, the proxy change is only for SaaS
2. yes that is fine, we should just not point users there

File: examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb [hunk 1]
```diff
@@ -241,64 +250,25 @@
...
-    "\n",
     "%%bash\n",
-    "curl \"https://api.wandb.ai/proxy/openai/v1/chat/completions\" \\\n",
+    "curl \"https://proxy.wandb.ai/proxy/openai/v1/chat/completions\" \\\n",
     "-H \"Authorization: Bearer $WANDB_API_KEY:$OPENAI_API_KEY\" \\\n",
     "-H \"Content-Type: application/json\" \\\n",
-    "-H \"X-Wandb-Entity: WB_ENTITY\" \\\n",
-    "-H \"X-Wandb-Project: WB_PROJECT\" \\\n",
-    "-H \"X-Wandb-Stream: STREAM_NAME\" \\\n",
+    "-H \"X-Wandb-Entity: stacey\" \\\n",
```
Comment (tssweeney) [inline examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb:257]: can you put the vars back here?

File: examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb [hunk 2]
```diff
@@ -241,64 +250,25 @@
...
+    "-H \"X-Wandb-Stream: proxy_stream\" \\\n",
     "-H \"X-Wandb-Client-Id: Shakespearean-Gradients-001\" \\\n",
     "-H \"X-Wandb-Attribute-Persona: Shakespeare\" \\\n",
     "-d '{\n",
     "     \"model\": \"gpt-3.5-turbo\",\n",
     "     \"messages\": [{\"role\": \"user\", \"content\": \"Tell me a joke about loss functions!\"}]\n",
     "   }'"
    ]
-  },
-  {
```
Comment (tssweeney) [inline examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb:262]: why remove?

**Review by tssweeney — COMMENTED** [review]
can we update the part like `4. Change the OpenAI base url to `https://wandb.ai/proxy/openai/v1` for`

File: examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb (hunk 2 above)
Comment (jlzhao27) [reply examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb:262]: https://weightsandbiases.slack.com/archives/C03BSTEBD7F/p1695923749471649?thread_ts=1695922367.393609&cid=C03BSTEBD7F

Comment (jlzhao27) [reply examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb:262]: the issue is we have a check in weave for `WANDB_API_KEY` being set AND `.netrc` exists. However, for proxy when netrc exists and they use `proxy.wandb.ai`, they need `WANDB_API_KEY` or else there is no auth. But weave prevents this request from going through.

Comment (tssweeney) [reply examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb:262]: why do they have netrc in their colab?!

**Review by tssweeney — APPROVED** [review]

**Review by jlzhao27 — APPROVED** [review]

**Pull request merged**